display_information/display-annotation/build.gradle

```diff
@@ -0,0 +1,78 @@
+plugins {
+    id 'application'
+    id 'org.openjfx.javafxplugin' version '0.0.5'
+}
+
+group = 'com.esri.samples'
+
+ext {
+    arcgisVersion = '100.8.0-2694'
```

Comment:
is this functionality released for 100.7.0, or should this be being merged into the dev branch?

Reply:
I originally thought it's 100.8.0, but you're right, it's 100.7.0. Changed!